build/build.proj

```diff
@@ -226,7 +226,18 @@
              Projects="@(SolutionProjects)"
              Targets="PackProjects"
              Properties="$(CommonMSBuildProperties);
-                         VisualStudioVersion=$(VisualStudioVersion);" />
+                         VisualStudioVersion=$(VisualStudioVersion);
+                         Create2ndNupkg=false" />
```

Comment:
Create2ndNupkg=false doesn't add any value. You don't need to set it.

Reply:
Thanks! Removed accordingly.